src/components/templates/Search/filterPrice.module.css

```diff
@@ -0,0 +1,19 @@
+.filterList {
+  display: inline-flex;
+  float: left;
```

Comment:
we usually try to avoid using `float` for any layout, but also the isolated component should not define its placement within its parent, to keep it flexible and isolated.

This can be achieved by defining the layout on the parent of sorting component & filtering component with e.g. `display: flex; justify-content: space-between`